src/devices/namron.ts

```diff
@@ -96,6 +111,80 @@ const tzLocal = {
             }
         },
     } satisfies Tz.Converter,
+
+    namronPlugg: {
```

Comment:
Instead of adding from/toZigbee and exposes, can you use the new `modernExtend` instead? Example https://github.com/Koenkk/zigbee-herdsman-converters/blob/9658b12fb629735dd2b57647d443be1b6b8b67c5/src/devices/sonoff.ts#L559

Reply:
I'll have a look at it, but I have never programmed in java script before, but I am still learning :-) , so it might take some time